internal/receiver/databricksreceiver/databricks_unmarshaller.go

```diff
@@ -0,0 +1,67 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package databricksreceiver
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// databricksUnmarshaller wraps a databricksClientIntf implementation and unmarshals json byte
+// arrays to the types defined in json_types.go. Its methods signatures mirror
+// those of the dbc.
+type databricksUnmarshaller struct {
```

Comment:
I am not sure if it's even worth mentioning, but the code uses marshaler and marshaller in places, do you care to pick one?